src/ontodia/diagram/elementLayerOwl.tsx

```diff
@@ -0,0 +1,289 @@
+import * as React from 'react';
+import { findDOMNode } from 'react-dom';
+import * as joint from 'jointjs';
+import * as Backbone from 'backbone';
+import { hcl } from 'd3-color';
+
+import { Property } from '../data/model';
+
+import { TemplateProps } from '../customization/props';
+
+import { Element } from './elements';
+import { uri2name } from './model';
+import { DiagramView } from './view';
+
+export interface Props {
+    paper: joint.dia.Paper;
+    view: DiagramView;
+}
+
+export class ElementLayerOwl extends React.Component<Props, void> {
```

Comment:
Do you really need to clone this class? Maybe there is some other way?

Reply:
No, I'll try differently.

Reply:
done without clone class ElementLayer